openesdh-repo/src/main/amp/config/alfresco/extension/templates/webscripts/dk/openesdh/documents/categories/doccategory.delete.desc.xml

```diff
@@ -0,0 +1,9 @@
+<webscript>
+    <shortname>Delete document category</shortname>
+    <description>Deletes document category</description>
+    <url>/api/openesdh/document/category?nodeRefId=</url>
+    <format default="json"></format>
+    <authentication>user</authentication>
```

Comment:
I suppose the authentication should be 'admin' to prevent categories deletion by anyone.